digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/util/DataFixture.scala

```diff
@@ -1067,6 +1067,111 @@ object DataFixture {
+
+      //Get all OTH Bus Stops By Municipality
+      persistedStop = massTransitStopService.getByMunicipality(municipality, false)
+
+      persistedStop.foreach { stop =>
+        val stopLiviId = stop.propertyData.
+          find(property => property.publicId == MassTransitStopOperations.LiViIdentifierPublicId).
+          flatMap(property => property.values.headOption).map(p => p.propertyValue)
+
+        // Validate if OTH stop are known in Tierekisteri and if is maintained by ELY
+        if (stopLiviId.isDefined && liviIdsListTR.contains(stopLiviId.get)) {
+          //Data From OTH
+          val stopNameSE =
+            stop.propertyData.find(property => property.publicId == MassTransitStopOperations.nameSePublicId).
+              flatMap(property => property.values.headOption).map(p => p.propertyValue)
+            match {
+              case Some(roofValue) => roofValue
+              case _ => ""
+            }
+
+          val stopRoofValue =
+            stop.propertyData.find(property => property.publicId == MassTransitStopOperations.roofPublicId).
+              flatMap(property => property.values.headOption).map(p => p.propertyValue)
+            match {
+              case Some(roofValue) => Existence.fromPropertyValue(roofValue)
+              case _ => ""
+            }
+
+          val stopRaisedBusStopValue =
+            stop.propertyData.find(property => property.publicId == MassTransitStopOperations.raisePublicId).
+              flatMap(property => property.values.headOption).map(p => p.propertyValue)
+            match {
+              case Some(raisedValue) => Existence.fromPropertyValue(raisedValue)
+              case _ => ""
+            }
+
+          //Data From TR
+          val busStopsTR = allBusStopsTR.find(_.liviId == stopLiviId.get)
+
+          val nameSEinTR = busStopsTR.head.nameSe match {
+            case Some(name) => name
+            case _ => ""
+          }
+          val roofValueinTR = busStopsTR.head.equipments.get(Equipment.Roof) match {
+            case Some(roofValue) => roofValue
+            case _ => ""
+          }
+          val raisedValueinTR = busStopsTR.head.equipments.get(Equipment.RaisedBusStop) match {
+            case Some(raisedValue) => raisedValue
+            case _ => ""
+          }
+
+          if ((stopNameSE != nameSEinTR) || (stopRoofValue != roofValueinTR) || (stopRaisedBusStopValue != raisedValueinTR)) {
+            val propertiesToUpdate = Seq(
+              SimpleProperty(MassTransitStopOperations.nameSePublicId, Seq(PropertyValue(nameSEinTR))),
+              SimpleProperty(MassTransitStopOperations.roofPublicId, Seq(PropertyValue(roofValueinTR.asInstanceOf[Existence].propertyValue.toString()))),
+              SimpleProperty(MassTransitStopOperations.raisePublicId, Seq(PropertyValue(raisedValueinTR.asInstanceOf[Existence].propertyValue.toString())))
+            )
+
+            massTransitStopService.updatePropertiesForAsset(stop.id, propertiesToUpdate)
```

Comment:
Check if it's possible to don't have in OTH side the property created.

Reply:
Already tested and if we create a Bus Stop without that Porperties they are added in the process.